recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ build:
 requirements:
   build:
     - python
+    - setuptools
```

Comment:
Needs these [arguments]( https://github.com/conda-forge/staged-recipes/blob/1caddd635ce1a2fbe477bc216878c8ea482e7cf1/recipes/example/meta.yaml#L26 ) when using `setuptools`. Can fix in a build number bump.